src/lib/gameService.ts

```diff
@@ -12,41 +13,172 @@ export class GameService {
 			code: this.generateCode(),
 			currentPlayer: undefined,
 			players: [],
-			currentBid: undefined
+			currentBid: undefined,
+			maxPlayers: 8,
+			numberOfPlayers: 0,
+			initialDiceCount: 6
 		};
 
 		this.repository.saveGame(game);
 
 		return game.code;
 	}
 
-	public async addPlayer(name: string, gameCode: string): Promise<string> {
-		throw new Error('not implemented');
+	public async addPlayer(playerName: string, gameCode: string): Promise<string> {
+		const game: Game | undefined = await gameRepository.getGame(gameCode);
```

Comment:
Make sure you're using the `this.repository` that was passed in to the constructor